### PR TITLE
Install/Upgrade Proxy Migration Improvements - jcayouette

### PR DESCRIPTION
**Pull request by jcayouette**
# Proxy Migration

## Task
- [ ] Add Proxy migration task to docs (SUMA / Uyuni)

## Branches
- master

# Links
- https://github.com/SUSE/spacewalk/issues/23471#issuecomment-2018269415

### CODE REVIEW

**Review by 0rnela — APPROVED** [review]

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+. Select btn:[Choose File] and browse your local machine for the certificate.
+. Select btn:[Choose File] and brose your local machine for the private key.
+. Enter the CA password.
+. Click btn:[Generate].
+
+=== Transfer the Proxy Configuration to the New Proxy Host
+
+.Task: Transfer the Proxy Configuration
+// Add details on this transfer, filename and where it is located. `/root/ssl-build/`
```
Comment (nodeg) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:45]: The layout for this is broken when being rendered:
image

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+.
```
[the remark this answers is not in the document]
Comment (nodeg) [reply modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:46]: The broken layout could be due to this extra dot.

**Comment by vandabarata** [comment]
> As a first version, this looks quite good. However, we should keep in mind to not only explain how we can migrate a 4.3 containerized proxy, but also a regular 4.3 proxy to 5.0.

Yes, that is _really_ important, in my opinion. And that should also be covered here

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+----
+proxy-salt-broker
+proxy-httpd
+proxy-tftpd
+proxy-squid
+proxy-ssh
+----
+
+
+
```
Comment (vandabarata) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:88]: After this, user still needs to migrate from systemd to mgrpxy - https://github.com/SUSE/spacewalk/issues/23471
image

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:46]: Known issue this section is still in progress. Thanks.

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 4]
```diff
@@ -84,5 +80,44 @@ proxy-squid
 proxy-ssh
 ----
 
+== Migrating a {productname} 4.3 Proxy to a {productname} {productnumber} Containerized Proxy
+
+. Boot your new machine and begin installation of {micro} {microversion}.
+. During the installation register with {scc} using your {micro} {microversion} registration code.
+. Next register this machine using your {productname} {productnumber} Proxy Extension code.
```
Comment (vandabarata) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:87]:
```suggestion
. Then register this machine using your {productname} {productnumber} Proxy Extension code.
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+.Disable the Legacy Proxy
+----
+spacewalk-proxy stop
+----
+
+. Deploy the new Proxy with:
++ 
+
+----
+mgrpxy install podman config.tar.gz
```
Comment (vandabarata) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:64]:
```suggestion
systemctl start uyuni-proxy-pod
```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+----
+
```
Comment (vandabarata) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:66]:
```suggestion

. Enable the new Proxy with:
+ 

----
systemctl enable --now uyuni-proxy-pod
----

```

File: modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+proxy-ssh
+----
+
+
+
+== Migrate {productname} 4.3 Proxy to {productname} {productnumber} Containerized Proxy
+
+
+
+.Task: Migrate {productname} 4.3 Proxy to {productname} {productnumber} Containerized Proxy
```
Comment (vandabarata) [inline modules/installation-and-upgrade/pages/container-deployment/suma/proxy-migration.adoc:84]:
```suggestion
.Task: Migrate {productname} 4.3 Containerized Proxy to {productname} {productnumber} New Containerized Proxy
```

**Review by vandabarata — CHANGES_REQUESTED** [review]

**Pull request merged**